student-store-ui/src/components/ProductDetail/ProductDetail.jsx

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import axios from 'axios';
+
+function ProductDetails() {
+  const { id } = useParams();
+  const [product, setProduct] = useState(null);
+
+  useEffect(() => {
+    axios.get(`https://codepath-store-api.herokuapp.com/store/${id}`)
+      .then(response => {
+        console.log(response.data.product);
+        setProduct(response.data.product);
+      });
+  }, []);
+
+  if (!product) {
+    return <div>Loading...</div>
+  }
```

Comment:
Nice use of a loading message for UX while waiting for the products from the API